python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
+
+def _get(self, key):
```

Comment:
There is no way to directly extend a C++ class in python, so we have to define a function and then assign it to the class. This is how we've been doing all the modules but if you know a better way I'd be happy to implement it.